### DISCUSSION TITLE
Downward reserves with thermal multistart formulation

### DISCUSSION THREAD

**Discussion by kdayday** [Sienna-Platform/PowerSimulations.jl#687] [General]
@jd-lara @sourabhdalvi 

I think there is an equation missing from pglib and the "Tight and Compact" formulation for downward reserves, because both assume the reserves are upward only. The equation should ensure downward reserves are supplied only when the unit is on. In the attached screenshots, RIO_NOGALES_POWER_CC4 is providing the downward reserves even though it is not on. I believe we just need to add: p_g,t - regdn_g,t >= 0 
where p_g,t is power above minimum.

![Screenshot 2021-05-18 202129](https://user-images.githubusercontent.com/12451220/118747204-cfcc5c00-b816-11eb-971e-184f4efdc563.png)
![Screenshot 2021-05-18 202202](https://user-images.githubusercontent.com/12451220/118747218-d8bd2d80-b816-11eb-8e32-a548c551db7f.png)

**Comment by jd-lara** [comment]
This is correct since in the compact formulation p is the value above min then an additional constraint is needed for downwards reserves to avoid weird behavior. What we need to check is if this is an issue elsewhere in the implementation of downwards reserves or just a missing constraint.
